### PR TITLE
Rewriting of readme

### PR DESCRIPTION
**Pull request by BuildGayFromSource**
Added explanations for individual cards and the header. I also added intermediary explanatory images, I will eventually update and standardize them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,85 @@
...
-Gjallarhorn is a tool for Brawlhalla community tournament runners to generate
-properly formatted JSON for Twitch overlays. This supports overlays for Game,
-Players, Queue, Ticker, Casters, and Lower Thirds. It pulls data from start.gg
-as well as Brawlhalla's Stats API.
+Gjallarhorn is a tool for Brawlhalla community tournament runners to generate properly formatted JSON for Twitch overlays. Gjallarhorn can generate JSON for multiple use cases, including tickers, casters, lower thirds, queues, players, and games. It pulls data from start.gg as well as Brawlhalla's Stats API.
 
-## Usage
+## Getting Started
+
+You will need a Start.gg API key to use Gjallarhorn. If you don't have one already, get one at https://developer.start.gg/docs/authentication.
```
Comment (preyneyv) [inline README.md]: probably doesn't matter but sgg's brand guidelines say we should only ever write "start.gg"
https://help.start.gg/en/articles/1716774-start-gg-brand-guidelines

**Review by preyneyv — COMMENTED** [review]
Looks great! Idk if it's worth mentioning shell customization like reordering, recoloring, and hiding cards, but otherwise a great summary!

File: README.md (hunk 1 above)
Comment (BuildGayFromSource) [reply README.md]: This is a good catch. I went ahead and standardized that usage in the doc.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,95 @@
...
-Check out https://developer.start.gg/docs/authentication to get started with an
-API key if you don't have one already. You can fill out the cards with the
-proper details you want on the overlay, and hit "Push". You will find the output
-files served over http at http://localhost:3000/api/json or the raw json files
-in `packages/cli/output`
+Then, launch the dashboard at http://localhost:3000.
+
+## Usage
+
+You will find 6 different cards on the dashboard: Casters, Game, Ticker, Players, Queue, and Lower Thirds. You can fill out the cards with the details you want on the overlay and hit "Push". You will find the output files served over http at http://localhost:3000/api/json or the raw JSON files in `packages/cli/output`.
```
Comment (arnabp) [inline README.md:21]: Either the names of each card here should link to that respective section the documentation below (like [this](https://github.com/giovcandido/sedona?tab=readme-ov-file#contents)), or we should put the sections for the cards in expandable sections (like [this](https://github.com/AntonioFalcaoJr/EventualShop?tab=readme-ov-file#mag_right-research))

File: README.md [hunk 3]
```diff
@@ -1,24 +1,95 @@
...
+
+![The Player card being reordered](images/gjallardrag.png)
+
+You can also recolor each card by right clicking on the title bar.
+
+![The Lower Thirds card with the color modal open](images/gjallarcolors.png)
+
+If a card is unnecessary or temporarily unneeded, you can minimize it by clicking the arrow at the right of each card next to the push button.
+
+These JSON files can be connected to broadcasting software (vMix or OBS with applicable plugins) to allow for seamless editing and usage of data. However, not all of this data will be useful or even applicable even if you are hosting a tournament.
```
Comment (arnabp) [inline README.md]: Adding something like "You are free to use or ignore whatever data fits your use case when linking the data in your broadcasting software" at the end here will tie it together nicely I think

File: README.md [hunk 4]
```diff
@@ -1,24 +1,95 @@
...
+
+### Header
+
+In the top left corner, the current tournament is indicated next to the hamburger menu. The pencil allows you to change the active tournament. Enter the tournament’s unique slug (“brawlhalla-world-championship-expo-2023”) to switch to that tournament. Shorthand forms also exist for certain tournaments (“bcx”). Clicking on the game mode (In this example, Brawlhalla Championship 1v1) allows you to swap between different tournament game modes if they exist. The pools can also be swapped similarly.
```
Comment (arnabp) [inline README.md]: *pencil icon

Change "swap(ped)" to "change" or "switch". I think swap has a connotation that implies the thing you're taking out is goes in place of the thing you're putting in when really all we're concerned about is what's going in.

File: README.md [hunk 5]
```diff
@@ -1,24 +1,95 @@
...
+
+Push Brackets will output JSON to be used in bracket images. You can have these pushed every two minutes by clicking the timer next to the button.
```
Comment (arnabp) [inline README.md]: output *a JSON

File: README.md [hunk 6]
```diff
@@ -1,24 +1,95 @@
...
+
+![An active user icon](images/gjallaruser.png)
+
+Hovering over this icon will show checkboxes that enable light mode and push notifications for possible errors respectively.
+
+### Casters
+
+![The Casters card](images/gjallarcasters.png)
+
+You can enter up to four different casters, with text boxes for their name, Twitter/X accounts, and pronouns.
```
Comment (arnabp) [inline README.md]: Let's just say social media accounts so we don't have to change this when twitter inevitably dies and we never have to think about it again

File: README.md [hunk 7]
```diff
@@ -1,24 +1,95 @@
...
+
+![The Player card](images/gjallarplayers.png)
+
+This card can be input manually. You can also import players from the Game card or the Queue card with the clipboard button next to Push. When importing players, Gjallarhorn will attempt to populate the Lifetime Score textbox with information from the Stats API. It can also be manually incremented with the plus and minus buttons. The legend loaded for each player will be the most recently reported legend in the current tournament (if applicable). Otherwise, it will be the last legend the player ever reported.
+
+### Ticker
+
+![The Ticker Card](images/gjallarticker.png)
+
+Can be used for miscellaneous ticker text. By clicking the timer next to shuffle, the tickers will automatically shuffle every five minutes.
```
Comment (arnabp) [inline README.md]: No reason to call the ticker text miscellaneous, we could have instances where tournament producers would consider the ticker info extremely non-miscellaneous.

**Review by arnabp — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (preyneyv) [reply README.md:21]: EXPANDABLE SECTIONS??? when did they update markdown

File: README.md (hunk 7 above)
Comment (preyneyv) [reply README.md]: Miscellaneous is the perfect word wym

image

File: README.md (hunk 2 above)
Comment (BuildGayFromSource) [reply README.md:21]: I did the former, but I'm leaving this as unresolved for the time being to see if I can't figure out expandable sections.

File: README.md (hunk 7 above)
Comment (BuildGayFromSource) [reply README.md]: Went ahead and changed it anyway, while also adding a bit more information to the card. Not sure if miscellaneous is important enough to split hairs over.

File: README.md (hunk 2 above)
Comment (BuildGayFromSource) [reply README.md:21]: The doc is in expandable sections now. I left the Project Overview and Contributors mostly untouched.

File: README.md [hunk 8]
```diff
@@ -1,24 +1,119 @@
...
+
+If a card is unnecessary or temporarily unneeded, you can minimize it by clicking the arrow at the right of each card next to the push button.
+
+These JSON files can be connected to broadcasting software (vMix or OBS with applicable plugins) to allow for seamless editing and usage of data. However, not all of this data will be useful or even applicable to all organizers, and you can decide what data you'd like to use. Feel free to disregard any of the forms or data that are unnecessary for your production environment.
+
+</details>
+
+<details>
+  <summary>Header</summary>
+
```
Comment (arnabp) [inline README.md:41]: Add a picture of the header here, since all the other categories have one

File: README.md [hunk 9]
```diff
@@ -1,24 +1,119 @@
...
+The Lower Thirds card has five sections: Twitter, Twitch, Message, Champion, and Preshow.
+
+Champion can be autofilled by clicking the autofill button. It will use the tournament name along with the game mode, and the winner’s name will be in the message box.
+
+You can create presets for each section. You can also delete the most recently created preset if it was added in error.
+
+</details>
 
 ## Project Overview
```
Comment (arnabp) [inline README.md]: Can you rename this to Development Overview

**Review by arnabp — APPROVED** [review]

**Pull request merged**